src/pages/Home/Home.scss

```diff
@@ -0,0 +1,32 @@
+@use "../../styles/variables" as *;
+
+.homepage {
```

Comment:
Can you nest your styles and use the `&` to match the other components in the project, please?